src/main/java/org/terasology/climateConditions/BodyTemperatureComponent.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.climateConditions;
+
+import org.terasology.entitySystem.Component;
+import org.terasology.network.Replicate;
+
+public class BodyTemperatureComponent implements Component {
+
+    @Replicate
```

Comment:
why do you need to replicate these values?

Reply:
Since we are considering multiple players, in the docs it was written, the information is not automatically updated on the server..... and to let all other players and server know, we need to use replicate....and well  this I felt is necessary for the part of the code where you've commented about the use of WorldComponent(onTemperatureUpdate).